hulks/base.py

```diff
@@ -26,8 +26,13 @@ def handle(self, args=None):
         return int(not retval)
 
     def lines_iterator(self, filename):
-        with open(filename) as fp:
-            for line_number, line in enumerate(fp.readlines(), 1):
-                # heuristic, so we dont need to handle all "comment" syntax accross languages
-                if ' noqa' not in line.lower():
-                    yield line_number, line
+        try:
+            with open(filename) as fp:
+                for line_number, line in enumerate(fp.readlines(), 1):
+                    # heuristic, so we dont need to handle all "comment" syntax accross languages
+                    if ' noqa' not in line.lower():
+                        yield line_number, line
+        except UnicodeDecodeError as error:
+            *args, reason = error.args
+            reason += ' at file {!r}!'.format(filename)
+            raise UnicodeDecodeError(*(args + [reason]))
```

Comment:
:lipstick: prefiro do jeito abaixo, pois deixa o código mais limpo já que deixa o código sem tantos ()[]

```suggestion
            raise UnicodeDecodeError(*args, reason)
```

Reply:
mas a partir de qual versão funciona isso? O @daneoshiga sugeriu eu retirar coisas de Py 3.6+  pq. esse linter pode rodar em servidores com python mais antigo.

Reply:
ok - testei aqui no py3.5, funciona.